config/policies.json

```diff
@@ -31,8 +31,8 @@
         "module": "$import(@zuplo/runtime)",
         "options": {
           "allowUnauthenticatedRequests": false,
-          "auth0Domain": "josh-demo.us.auth0.com",
-          "audience": "https://zuplo-consumer-api"
+          "auth0Domain": "$env(AUTH0_DOMAIN)",
```

Comment:
@ntotten we should hard code this to the demo tenant we'll use eh?

Reply:
yeah, this one is tricker i realized. If we hard code this here it makes it even harder to setup the env variable. We don't have a fallback option on this syntax unfortunately. Proably another reason we should have used `${env.AUTH0_DOMAIN ?? "fallback"}`